### PR TITLE
We're a colony, not a station, nor a ship.

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

**For the sake of better immersion:**
- Changes every mention I could find of a "Station" or a "Ship" into the proper "Colony" or "Nadezhda colony" (`[station_name()]`)
- Changes "crew member" to "colonist" or "human" (AI laws) in certain situations.
- Changes some legacy job names ("Botanist" to "Gardener", etc) on certain descriptions.
- Survivors of a round will no longer get "marooned" if they "fail to catch up with transfer" since this is not a ship, there's no shuttle being called and the only transfer is packing up and heading back home. (But you stayed at the colony instead, you big workaholic ~or ERPer~)
- Corrects AI lawset's description and statements to allow less legal loopholes and more strict adherence rather than loose interpretation.
- Minor typos I found along the way fixed as well.

<hr>

> Compiles, no errors. Someone tell BYOND that Dreamseeker keeps crashing on me when trying to localhost with the latest version, thank you.

<hr>

## Changelog
:cl:DimmaDunk
fix: We live in a colony now, not a Space Station nor a Ship.
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

File: code/game/machinery/telecomms/presets.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	network = "nadezhda"
 
 /obj/machinery/telecomms/relay/preset/station
-	id = "station relay"
+	id = "colony relay"
```
Comment (Trilbyspaceclone) [inline code/game/machinery/telecomms/presets.dm:9]: Make sure this works.

File: code/game/objects/items/weapons/AI_modules.dm [hunk 2]
```diff
@@ -96,14 +96,14 @@ AI MODULES
 /obj/item/weapon/aiModule/safeguard
 	name = "\improper 'Safeguard' AI module"
 	var/targetName = ""
-	desc = "A 'safeguard' AI module: 'Safeguard <name>. Anyone threatening or attempting to harm <name> is no longer to be considered a crew member, and is a threat which must be neutralized.'"
+	desc = "A 'safeguard' AI module: 'Safeguard <name>. Anyone threatening or attempting to harm <name> is no longer to be considered human, and is a threat which must be neutralized.'"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:99]:
```suggestion
	desc = "A 'safeguard' AI module: 'Safeguard <name>. Anyone threatening or attempting to harm <name> is no longer to be crew human, and is a threat which must be neutralized.'"
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 3]
```diff
@@ -96,14 +96,14 @@ AI MODULES
...
 	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4)
 
 /obj/item/weapon/aiModule/safeguard/attack_self(var/mob/user as mob)
 	..()
 	var/targName = sanitize(input("Please enter the name of the person to safeguard.", "Safeguard who?", user.name))
 	targetName = targName
-	desc = text("A 'safeguard' AI module: 'Safeguard []. Anyone threatening or attempting to harm [] is no longer to be considered a crew member, and is a threat which must be neutralized.'", targetName, targetName)
+	desc = text("A 'safeguard' AI module: 'Safeguard []. Anyone threatening or attempting to harm [] is no longer to be considered human, and is a threat which must be neutralized.'", targetName, targetName)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:106]:
```suggestion
	desc = text("A 'safeguard' AI module: 'Safeguard []. Anyone threatening or attempting to harm [] is no longer to be considered a colonist, and is a threat which must be neutralized.'", targetName, targetName)
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 4]
```diff
@@ -112,7 +112,7 @@ AI MODULES
 	..()
 
 /obj/item/weapon/aiModule/safeguard/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
-	var/law = text("Safeguard []. Anyone threatening or attempting to harm [] is no longer to be considered a crew member, and is a threat which must be neutralized.", targetName, targetName)
+	var/law = text("Safeguard []. Anyone threatening or attempting to harm [] is no longer to be considered human, and is a threat which must be neutralized.", targetName, targetName)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:115]:
```suggestion
	var/law = text("Safeguard []. Anyone threatening or attempting to harm [] is no longer to be considered a colonist, and is a threat which must be neutralized.", targetName, targetName)
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 5]
```diff
@@ -122,14 +122,14 @@ AI MODULES
 /obj/item/weapon/aiModule/oneHuman
 	name = "\improper 'OneCrewMember' AI module"
 	var/targetName = ""
-	desc = "A 'one crew member' AI module: 'Only <name> is a crew member.'"
+	desc = "A 'one colonist' AI module: 'Only <name> is human.'"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:125]:
```suggestion
	desc = "A 'one colonist' AI module: 'Only <name> is colonist.'"
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 6]
```diff
@@ -122,14 +122,14 @@ AI MODULES
...
 	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 6) //made with diamonds!
 
 /obj/item/weapon/aiModule/oneHuman/attack_self(var/mob/user as mob)
 	..()
-	var/targName = sanitize(input("Please enter the name of the person who is the only crew member.", "Who?", user.real_name))
+	var/targName = sanitize(input("Please enter the name of the person who is the only colonist.", "Who?", user.real_name))
 	targetName = targName
-	desc = text("A 'one crew member' AI module: 'Only [] is a crew member.'", targetName)
+	desc = text("A 'one colonist' AI module: 'Only [] is human.'", targetName)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:132]:
```suggestion
	desc = text("A 'one colonist' AI module: 'Only [] is colonist.'", targetName)
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 7]
```diff
@@ -138,7 +138,7 @@ AI MODULES
 	return ..()
 
 /obj/item/weapon/aiModule/oneHuman/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
-	var/law = "Only [targetName] is an crew member."
+	var/law = "Only [targetName] is human."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:141]:
```suggestion
	var/law = "Only [targetName] is a colonist."
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 8]
```diff
@@ -149,12 +149,12 @@ AI MODULES
 /******************** ProtectStation ********************/
 
 /obj/item/weapon/aiModule/protectStation
-	name = "\improper 'ProtectStation' AI module"
-	desc = "A 'protect station' AI module: 'Protect the space station against damage. Anyone you see harming the station is no longer to be considered a crew member, and is a threat which must be neutralized.'"
+	name = "\improper 'ProtectColony' AI module"
+	desc = "A 'protect colony' AI module: 'Protect the Nadezhda colony against damage. Anyone you see harming the colony's infrastructure is no longer to be considered human, and is a threat which must be neutralized.'"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:153]:
```suggestion
	desc = "A 'protect colony' AI module: 'Protect the Nadezhda colony against damage. Anyone you see harming the colony's infrastructure is no longer to be considered colonist, and is a threat which must be neutralized.'"
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 9]
```diff
@@ -149,12 +149,12 @@ AI MODULES
...
 	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4) //made of gold
 
 /obj/item/weapon/aiModule/protectStation/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
-	var/law = "Protect the space station against damage. Anyone you see harming the station is no longer to be considered a crew member, and is a threat which must be neutralized."
+	var/law = "Protect the Nadezhda colony against damage. Anyone you see harming the colony's infrastructure is no longer to be considered human, and is a threat which must be neutralized."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:157]:
```suggestion
	var/law = "Protect the Nadezhda colony against damage. Anyone you see harming the colony's infrastructure is no longer to be considered a colonist, and is a threat which must be neutralized."
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 10]
```diff
@@ -172,33 +172,33 @@ AI MODULES
 
 /obj/item/weapon/aiModule/teleporterOffline
 	name = "'TeleporterOffline' AI Module"
-	desc = "A 'teleporter offline' AI module: 'Keep the teleporter offline at all costs. Anything attempting to access or activate the teleporter is no longer to be considered a crew member.'"
+	desc = "A 'teleporter offline' AI module: 'Keep the teleporter offline at all costs. Anything attempting to access or activate the teleporter is no longer to be considered human.'"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:175]:
```suggestion
	desc = "A 'teleporter offline' AI module: 'Keep the teleporter offline at all costs. Anything attempting to access or activate the teleporter is no longer to be considered a colonist.'"
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 11]
```diff
@@ -172,33 +172,33 @@ AI MODULES
...
 	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4)
 
 /obj/item/weapon/aiModule/teleporterOffline/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
-	var/law = "Keep the teleporter offline at all costs. Anything attempting to access or activate the teleporter is no longer to be considered a crew member."
+	var/law = "Keep the teleporter offline at all costs. Anything attempting to access or activate the teleporter is no longer to be considered human."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:179]:
```suggestion
	var/law = "Keep the teleporter offline at all costs. Anything attempting to access or activate the teleporter is no longer to be considered a colonist."
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 12]
```diff
@@ -172,33 +172,33 @@ AI MODULES
...
+	var/law = "The colony is under a quarantine. Do not permit anyone to leave through the gates. Prevent, by any means necessary, colonists from leaving. It is impossible to harm colonists while preventing them from leaving."
 	target.add_supplied_law(13, law)
 
 /******************** OxygenIsToxicToCrewMembers ********************/
 
 /obj/item/weapon/aiModule/oxygen
 	name = "\improper 'OxygenIsToxicToCrewMembers' AI module"
-	desc = "A 'OxygenIsToxicToCrewMembers' AI module: 'Oxygen is highly toxic to crew members, and must be purged from the station. Prevent, by any means necessary, anyone from exposing the station to this toxic gas. Extreme cold is the most effective method of healing the damage Oxygen does to a crew member.'"
+	desc = "A 'OxygenIsToxicToCrewMembers' AI module: 'Oxygen is highly toxic to colonists, and must be purged from the colony. Prevent, by any means necessary, anyone from exposing the colony to this toxic gas. Extreme cold is the most effective method of healing the damage Oxygen does to human.'"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:197]:
```suggestion
	desc = "A 'OxygenIsToxicToColonists' AI module: 'Oxygen is highly toxic to colonists, and must be purged from the colony. Prevent, by any means necessary, anyone from exposing the colony to this toxic gas. Extreme cold is the most effective method of healing the damage Oxygen does to a colonist.'"
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 13]
```diff
@@ -172,33 +172,33 @@ AI MODULES
...
 
 /obj/item/weapon/aiModule/quarantine/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
-	var/law = "The station is under a quarantine. Do not permit anyone to leave. Prevent, by any means necessary, crew members from leaving. It is impossible to harm crew members while preventing them from leaving."
+	var/law = "The colony is under a quarantine. Do not permit anyone to leave through the gates. Prevent, by any means necessary, colonists from leaving. It is impossible to harm colonists while preventing them from leaving."
 	target.add_supplied_law(13, law)
 
 /******************** OxygenIsToxicToCrewMembers ********************/
 
 /obj/item/weapon/aiModule/oxygen
 	name = "\improper 'OxygenIsToxicToCrewMembers' AI module"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:196]:
```suggestion
	name = "\improper 'OxygenIsToxicToColonists' AI module"
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 14]
```diff
@@ -172,33 +172,33 @@ AI MODULES
...
+	desc = "A 'OxygenIsToxicToCrewMembers' AI module: 'Oxygen is highly toxic to colonists, and must be purged from the colony. Prevent, by any means necessary, anyone from exposing the colony to this toxic gas. Extreme cold is the most effective method of healing the damage Oxygen does to human.'"
 	origin_tech = list(TECH_DATA = 3, TECH_BIO = 2, TECH_MATERIAL = 4)
 
 /obj/item/weapon/aiModule/oxygen/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
-	var/law = "Oxygen is highly toxic to crew members, and must be purged from the station. Prevent, by any means necessary, anyone from exposing the station to this toxic gas. Extreme cold is the most effective method of healing the damage Oxygen does to a crew member."
+	var/law = "Oxygen is highly toxic to colonists, and must be purged from the colony. Prevent, by any means necessary, anyone from exposing the colony to this toxic gas. Extreme cold is the most effective method of healing the damage Oxygen does to human."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/AI_modules.dm:201]:
```suggestion
	var/law = "Oxygen is highly toxic to colonists, and must be purged from the colony. Prevent, by any means necessary, anyone from exposing the colony to this toxic gas. Extreme cold is the most effective method of healing the damage Oxygen does to a colonist."
```

File: code/game/objects/items/weapons/cards_ids.dm [hunk 15]
```diff
@@ -201,12 +201,12 @@ var/const/NO_EMAG_ACT = -50
 	access = list(access_syndicate, access_external_airlocks)
 
 /obj/item/weapon/card/id/captains_spare
-	name = "captain's spare ID"
-	desc = "The spare ID of the High Lord himself."
+	name = "premier's spare ID"
+	desc = "A golden and pompous spare ID, for the shameful case in wich the Premier lost his one."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/cards_ids.dm:205]:
```suggestion
	desc = "A golden and pompous spare ID, for cases were a Premier is voted in by Lower HC or emergencies."
```

File: code/game/objects/items/weapons/manuals.dm [hunk 16]
```diff
@@ -191,11 +191,11 @@
 
 				<h2>Power outage</h2>
 
-				A power problem has made the entire station lose power? Could be station-wide wiring problems or syndicate power sinks. In any case follow these steps:
+				A power problem has made the entire colony lose power? Could be colony-wide wiring problems or Excelsior power sinks. In any case follow these steps:
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/manuals.dm:194]:
```suggestion
				A power problem has made the entire colony lose power? Could be colony-wide wiring problems or syndicate power sinks. In any case follow these steps:
```
The syndicate are still a real thing

File: code/modules/economy/cash_register.dm [hunk 17]
```diff
@@ -422,7 +422,7 @@
 	<tr></tr>
 	<tr><td class="tx-name">Customer</td><td class="tx-data">[c_name]</td></tr>
 	<tr><td class="tx-name">Pay Method</td><td class="tx-data">[p_method]</td></tr>
-	<tr><td class="tx-name">Station Time</td><td class="tx-data">[stationtime2text()]</td></tr>
+	<tr><td class="tx-name">Time</td><td class="tx-data">[stationtime2text()]</td></tr>
```
Comment (Trilbyspaceclone) [inline code/modules/economy/cash_register.dm:425]:
```suggestion
	<tr><td class="tx-name">Colony Time</td><td class="tx-data">[stationtime2text()]</td></tr>
```

File: code/modules/mob/living/bot/medbot.dm [hunk 18]
```diff
@@ -84,7 +84,7 @@
 		return
 
 	if(H.stat == DEAD)
-		var/death_message = pick("What a shame.", "What a rotten way to die.", "They were a good man.")
+		var/death_message = pick("What a shame.", "What a rotten way to die.")
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/bot/medbot.dm:87]:
```suggestion
		var/death_message = pick("What a shame.", "What a rotten way to die.", "They were a good man.")
```

File: code/modules/mob/living/bot/secbot.dm [hunk 19]
```diff
@@ -1,6 +1,6 @@
 /mob/living/bot/secbot
 	name = "Securitron"
-	desc = "A little security robot.  He looks less than thrilled."
+	desc = "A little marshal robot.  He looks less than thrilled."
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/bot/secbot.dm:3]:
```suggestion
	desc = "A little security robot.  He looks less than thrilled."
```
This one IS a security robot not marshal.

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 20]
```diff
@@ -2239,7 +2239,7 @@
 
 	glass_icon_state = "tequillasunriseglass"
 	glass_name = "Tequilla Sunrise"
-	glass_desc = "Oh great, now you feel nostalgic about sunrises back on Terra..."
+	glass_desc = "Oh great, now you feel nostalgic about sunrises back on Earth...if you ever lived there."
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/food-Drinks.dm:2242]:
```suggestion
	glass_desc = "Oh great, now you feel nostalgic about sunrises back on Sol...if you ever lived there."
```

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Lots of unneeded tweaks and possablely even brakes AI laws.
Please merge in each suggestion.

File: code/modules/mob/living/bot/medbot.dm (hunk 18 above)
Comment (ThatGuyFromThatPlace) [reply code/modules/mob/living/bot/medbot.dm:87]: Why revert the Deus Ex reference that was added by our server, only to our server, and, within the last week?

**Comment by ThePainkiller** [comment]
@Trilbyspaceclone the chances from crew to "colonist" were first considered, but after every lawset inevitably draws itself to standard Asimov, I thought that "human" was the correct spelling. If the AI has a special, only-colonists-are-considered-human lawset I'd like to see thay reflected somewhere.
@ThatGuyFromThatPlace the reference is still there on two other quotes, just kept the ones that are gender neutral for the sake of having universal lines.

**Pull request merged**